wordpress-mobile/WordPress-Android
# Issue/11935 localization of local announcements

Fixes #11935 

This PR adds support for localized local announcements.

To test:
Before testing I do recommend commenting modifying logic of [this](https://github.com/wordpress-mobile/WordPress-Android/blob/issue/11935-localization-of-local-announcements/WordPress/src/main/java/org/wordpress/android/viewmodel/main/WPMainActivityViewModel.kt#L168) method, so the feature announcement will be shown every time on app start.

It should look something like this:
  ```
  private fun checkForFeatureAnnouncements() {
        val currentVersionCode = buildConfigWrapper.getAppVersionCode()

        if (featureAnnouncementProvider.isAnnouncementOnUpgradeAvailable()) {
            appPrefsWrapper.featureAnnouncementShownVersion =
                    featureAnnouncementProvider.getLatestFeatureAnnouncement()?.announcementVersion!!
            _onFeatureAnnouncementRequested.call()
        }
    }
```

Start by copying the content of `FEATURE_ANNOUNCEMENTS.json-example` into `FEATURE_ANNOUNCEMENTS.json`.

- While App language set to English, launch the app and confirm that the feature announcement is displayed. 
- Change app language to a different variation of English, like Canadian or UK, launch the app and confirm that the feature announcement is displayed. 
- Change app language to Italian, launch the app, and confirm that the feature announcement is displayed features Italian content.
- Open `FEATURE_ANNOUNCEMENTS.json` and remove Italian translation from one of the features. Launch the app and confirm that the feature announcement is not displayed. Change language to English, and confirm that feature announcement IS displayed when you restart the app.
- This PR also adds support for base64 image loading, so confirm that you see a green android icon for in the first feature.

Local announcements are temporary, and I don't want to overengineer them, so I assume that announcement will always have an `en` locale.


PR submission checklist:
